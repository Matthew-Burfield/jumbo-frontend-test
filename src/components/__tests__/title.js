import React from 'react'
import { render } from '@testing-library/react'
import Title from '../title'
describe('Title', () => {
  it('renders correctly', () => {
    const { container, getByText } = render(<Title searchTerm="" />)
    expect(container).toMatchSnapshot()
    expect(getByText('Popular')).toBeTruthy()
  })

  it('updates the title after having searched for a new movie', () => {
    const { container, getByText } = render(
      <Title searchTerm="The Lion King" />
    )
    expect(container).toMatchSnapshot()
    expect(getByText('Results: The Lion King')).toBeTruthy()
  })
})
