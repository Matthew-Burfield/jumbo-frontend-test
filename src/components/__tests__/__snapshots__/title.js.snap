// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title renders correctly 1`] = `
<div>
  <h1
    class="css-75jtew-Title"
  >
    Popular
  </h1>
</div>
`;

exports[`Title updates the title after having searched for a new movie 1`] = `
<div>
  <h1
    class="css-75jtew-Title"
  >
    Results: The Lion King
  </h1>
</div>
`;
